### PR TITLE
Add Installation Confirmation Section for Minikube

### PR DESCRIPTION
**Pull request by danielhelfand**
This pull request adds instructions for confirming `minikube` has been appropriately installed after downloading a hypervisor and `minikube`. It allows the user going through these docs a chance to confirm `minikube` is appropriately linked to the hypervisor.

### CODE REVIEW

**Comment by danielhelfand** [comment]
See that this is documented under a section on [Installing Kubernetes with Minikube](https://deploy-preview-17479--kubernetes-io-master-staging.netlify.com/docs/setup/learning-environment/minikube/#specifying-the-vm-driver). 

Perhaps what would help these instructions is linking to that information and then also including and instruction in the documentation here to confirm `minikube` is installed by running `minikube start` with the appropriate `--vm-driver`.

**Comment by danielhelfand** [comment]
@sftim Thanks for your review! I pushed up something that's a little bit different of an approach. It's basically a section on confirming the installation of `minikube` that references the `--vm-driver` documentation already available. Let me know your thoughts.

File: content/en/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -153,7 +153,11 @@ If you do not already have a hypervisor installed, install one of these now:
 • [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 
 {{< note >}}
+
+**Hyper-V**
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-minikube.md]: Maybe remove this line?

Comment (danielhelfand) [reply content/en/docs/tasks/tools/install-minikube.md]: Thanks, and sorry for not cleaning this up earlier. Removed with latest push.

**Comment by tengqm** [comment]
Emm ... a conflict bubbles up.

**Comment by danielhelfand** [comment]
@tengqm Minor conflict from #17506 that's been resolved. Thanks for pointing it out.

File: content/en/docs/tasks/tools/install-minikube.md [hunk 2]
```diff
@@ -186,6 +186,41 @@ To install Minikube manually on Windows, download [`minikube-windows-amd64`](htt
 
 {{% /capture %}}
 
+## Confirm Installation
+
+To confirm successful installation of both a hypervisor and `minikube`, you can run the following command to start up a local Kubernetes cluster:
+
+{{< note >}}
+
+For setting the `--vm-driver` with `minikube start`, enter the name of the hypervisor you installed in lowercase letters where `<driver_name>` is mentioned below. A full list of `--vm-driver` values is available in the [specifying the vm driver documentation](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-minikube.md]: nit:
```suggestion
For setting the `--vm-driver` with `minikube start`, enter the name of the hypervisor you installed in lowercase letters where `<driver_name>` is mentioned below. A full list of `--vm-driver` values is available in [specifying the VM driver](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
```

File: content/en/docs/tasks/tools/install-minikube.md [hunk 3]
```diff
@@ -186,6 +186,41 @@ To install Minikube manually on Windows, download [`minikube-windows-amd64`](htt
...
+`minikube status` should return the following output:
+
+```
+host: Running
+kubelet: Running
+apiserver: Running
+kubeconfig: Configured
+```
+
+The above output from `minikube` confirms `minikube` is working with your chosen hypervisor. To stop your cluster, run the following command:
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-minikube.md]:
```suggestion
After you have confirmed whether Minikube is working with your chosen hypervisor, you can continue to use Minikube or you can stop your cluster. To stop your cluster, run:
```

File: content/en/docs/tasks/tools/install-minikube.md [hunk 4]
```diff
@@ -186,6 +186,41 @@ To install Minikube manually on Windows, download [`minikube-windows-amd64`](htt
...
+minikube start --vm-driver=<driver_name>
+```
+
+Once `minikube start` finishes, run the command below to check the status of the cluster:
+
+```shell
+minikube status
+```
+
+`minikube status` should return the following output:
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-minikube.md]:
```suggestion
If your cluster is working OK, the output from `minikube status` is similar to:
```

**Comment by Rajakavitha1** [comment]
Hi @danielhelfand could you please incorporate the suggestions that @sftim made so that we can proceed with the approval of this PR.

**Comment by danielhelfand** [comment]
@Rajakavitha1 @sftim Just pushed up requested changes. Thanks!

**Comment by Rajakavitha1** [comment]
Thanks @danielhelfand !!!
/lgtm
/approve

**Pull request merged**